### PR TITLE
😎 - Redesign the Sign In components

### PR DESCRIPTION
**Pull request by benjdlambert**
### Problem
- The Auth0 lock is boring 😴
- We need more flexibility

### Solution
- Build our own using React
- Make it look like giphy.com/login

screen shot 2016-09-08 at 13 09 38
screen shot 2016-09-08 at 13 09 20

### Notes
@revett @stevenjack

### CODE REVIEW

**Comment by benjdlambert** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;812ieylP&#x2F;381-cr-br-reusable-login-page"> [CR] [BR] Reusable Login page</a>

**Comment by benjdlambert** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;xVE5YREz&#x2F;382-cr-br-reusable-signup-page"> [CR] [BR] Reusable Signup page</a>

**Comment by benjdlambert** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;rKn4E9CD&#x2F;383-cr-br-reusable-forgot-password-page"> [CR] [BR] Reusable Forgot Password page</a>

File: Gulpfile.js [hunk 1]
```diff
@@ -81,10 +83,15 @@ function getFolders(dir) {
...
   'build',
   [
-    'build:js', 'build:jsx', 'build:jsx:vendor', 'build:sass',
+    'build:js', 'build:jsx', 'build:jsx:vendor', 'build:sass', 'build:images'
   ]
 );
 
+gulp.task('build:images', () =>
+  gulp.src(paths.IMG_SRC)
+    .pipe($.imagemin())
```
Comment (revett) [inline Gulpfile.js]: Nice!

**Comment by revett** [comment]
🍰 

Looks good! Think everything will be caught on Staging w/ QA

**Pull request merged**